### ISSUE TITLE
Document example of using dask_gateway

### ISSUE THREAD

**Issue by consideRatio** [2i2c-org/infrastructure#2144]
We ought to provide some example on how to use `dask_gateway` in the `daskhub` we provide. This is an preliminary example that has run into issues tracked in #2145.

## Action points

- [ ] Resolve #2145
- [ ] Document this example somewhere
- [ ] Verify verifying this example works at ciroh.awi.2i2c.cloud
- [ ] Followup with Matt at ciroh.awi.2i2c.cloud in [this ticket](https://2i2c.freshdesk.com/a/tickets/383)

## Dask gateway example

This usage documentation is similar to [dask-gateway's official usage documentation](https://gateway.dask.org/usage.html), but is simpler as how to connect to the dask-gateway server is pre-configured via environment variables.

To use dask-gateway from a user server, the user server should have `dask-gateway` installed, which is a client to communicate with an external `dask-gateway-server`. The server should also have `bokeh`, as the image will be used in the scheduler pod that needs `bokeh` to present dashboards. At the time of writing, [only `bokeh` version 2 is supported](https://github.com/dask/distributed/issues/7414#issuecomment-1418767284).

```python
# Create a pre-configured dask-gateway client to speak with an already running
# dask-gateway server. With this client, you can request that the server creates
# create a dask cluster for you, which means you get one dask scheduler and your
# choice of dask workers to run separate from this server.
#
from dask_gateway import Gateway
gateway_client = Gateway()
```

```python
# Request options for you to configure on a dask cluster to be created.
#
cluster_options = gateway_client.cluster_options()
cluster_options
```

```python
# Use the options to create a dask cluster.
#
# After running this cell and a dask-scheduler has started up, you are presented
# with a control panel view to add/remove workers.
#
# Note that the scheduler and workers will run on a separate node/machine that
# likeley needs to start up. When a node needs to be started, expect ~5 minutes
# for it to startup so that the workers can run on it and become registered with
# the scheduler.
#
cluster = gateway_client.new_cluster(cluster_options)
cluster
```

```python
# dask-gateway has done its job, you now interact with the dask-scheduler of the
# created dask cluster using a dask scheduler client.
#
scheduler_client = cluster.get_client(set_as_default=True)
scheduler_client
```

## Example job to execute

```python
# This defines a dummy task that we could execute:
# - locally without Dask
# - locally using Dask and local workers (dedicated linux processes)
# - remotely using Dask and remote workers (dedicated Kubernetes Pods)
#
import numpy as np
import joblib
from sklearn.datasets import load_digits
from sklearn.model_selection import RandomizedSearchCV
from sklearn.svm import SVC

digits = load_digits()

param_space = {
    'C': np.logspace(-6, 6, 13),
    'gamma': np.logspace(-8, 8, 17),
    'tol': np.logspace(-4, -1, 4),
    'class_weight': [None, 'balanced'],
}

model = SVC(kernel='rbf')
search = RandomizedSearchCV(model, param_space, cv=3, n_iter=50, verbose=10)
```

```python
# Let's execute the dummy task by using our scheduler client configured to be
# used by default, which in turn delegates work to the clusters workers.
#
# This can for example take 12 seconds with 8 1-CPU workers.
#
with joblib.parallel_backend('dask'):
    search.fit(digits.data, digits.target)
```

**Comment by consideRatio** [comment]
Resolved by @jnywong in https://github.com/2i2c-org/docs/pull/228
